### ISSUE TITLE
ViroParticleEmitter has no propType for nativeProp VRTParticleEmitter.ignoreEventHandling of native type boolean

### ISSUE THREAD

**Issue by fadani** [viromedia/viro#211]
Hi!

After upgrading to viro 2.5.1 I get this error when I want to display the AR screen:

![screenshot_20180405-135511](https://user-images.githubusercontent.com/1943401/38369875-07b2b570-38e9-11e8-9c5f-1b67463458bb.png)

We don't use the `ignoreEventHandling` prop anywhere. 

Do you have any suggestion where to look for the problem?

Thank you!

**Comment by manbod** [comment]
Hi @fadani ,
Are you seeing this error on the Test Bed app? Or your own stand alone app? Right after the error - `ViroParticleEmitter has no propType for nativeProp VRTParticleEmitter.ignoreEventHandling of native type boolean`, React Native is showing a hint - `If you haven't changed this prop yourself, this ...` . That usually means your test bed app is not pointing to the latest update. Updating your test bed app from App Store / Play Store to the latest version should resolve this for you. Let us know if that doesn't work for you.

Thanks!

**Comment by achuvm** [comment]
Hi @fadani,

This actually looks like a bug on our end.

If you navigate to `node_modules/react-viro/components/ViroParticleEmitter.js`, you'll need to add the following line:

`ignoreEventHandling: true,`

To the list of "ignored" props at the bottom of the file:

```
var VRTParticleEmitter = requireNativeComponent(
  'VRTParticleEmitter', ViroParticleEmitter, {
    nativeOnly: {
      onNativeTransformDelegateViro:true,
      hasTransformDelegate:true,
      canHover: true,
      canClick: true,
      canTouch: true,
      ...
    }
}

```

This apparently only occurs in debug builds (release builds should be fine).

We'll fix this in our next release.

Thanks!

**Comment by dam00n** [comment]
This issue has been fixed in our latest release, ViroReact 2.6.0. Release notes and upgrade instructions here -> https://virocore.viromedia.com/docs/3d-objects#section-loading-3d-objects

Closing this issue. Please reopen if you still have issues.
